### ISSUE TITLE
Show on profile-page the date when joined nostr

### ISSUE THREAD

**Issue by dwebert** [hzrd149/nostrudel#70]
![image](https://github.com/hzrd149/nostrudel/assets/18017148/5479f565-fc70-4087-afc5-be7616d2d59a)

**Comment by hzrd149** [comment]
This is a cool feature but I'm not sure how to get that data. all relays return events in chronological order and you cant really ask for the oldest event :(

**Comment by dwebert** [comment]
Yeah you are right. Primal also just uses the oldest note.  I thought the date would accessible like in PGP
